### PR TITLE
Bump version to 1.1.0-rc3

### PR DESCRIPTION
**Pull request by vieux**
ping @dongluochen @abronan

### CODE REVIEW

**Comment by vieux** [comment]
ping @docker/swarm-maintainers

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 1.1.0 (2016-01-14)
+## 1.1.0 (2016-02-2)
```
Comment (dongluochen) [inline CHANGELOG.md]: nit: 2016-02-02

**Comment by dongluochen** [comment]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,8 +17,11 @@
 - Fix error when inspect on unhealthy node
 - Prevent panic in filters when container has no name
 - Add buildtime, kernelversion and experimental to API version
+- Support docker update and new networking related flags in run & network create/connect/disconnect/ls
+- Require --all on docker ps to display containers on unhealthy nodes
```
Comment (nishanttotla) [inline CHANGELOG.md]: nit: use markdown for `--all`?

**Comment by abronan** [comment]
LGTM

**Comment by vieux** [comment]
@dongluochen @nishanttotla updated

**Comment by nishanttotla** [comment]
LGTM

**Pull request merged**